install/roles/sogo/templates/sogo.conf

```
@@ -17,24 +20,26 @@
   SOGoAppointmentSendEMailNotifications = {{ sogo.appointment_send_emails }};
   SOGoEnableEMailAlarms = YES;
 
-  // Mail parameters
-  SOGoMailingMechanism = smtp;
-  // No authentication, as the server only supports it after STARTTLS
-  // and SOGo does not seem to support STARTTLS for SMTP
-  // SOGoSMTPAuthenticationType = plain;
-  SOGoSMTPServer = "smtp.{{ network.domain }}:{{ mail.postfix.submission.port }}";
-  SOGoTrashFolderName = Trash;
-  SOGoDraftsFolderName = Drafts;
+  // Mail SMTP parameters: use SOGo standards
+  SOGoSMTPAuthenticationType = plain;
+  SOGoSMTPServer = "smtp://smtp.{{ network.domain }}:587/?tls=YES";
```

```suggestion
  SOGoSMTPServer = "smtp://smtp.{{ network.domain }}:{{ mail.postfix.submission.port }}/?tls=YES";
```
If Sogo supports TLS with `tls=YES`, and not only STARTTLS, it could also use mail.postfix.submission**s**.port.